### PR TITLE
Added support for configuring manifest links of the downloaded manifest

### PR DESCRIPTION
**Pull request by kbidarkar**

### CODE REVIEW

File: automation_tools/__init__.py [hunk 1]
```diff
@@ -2532,3 +2538,20 @@ def download_manifest(url=None, session=None, distributor=None):
...
+
+
+def configure_manifest(manifest=None, prefix_dir=None):
+    """Configures the latest downloaded manifest file to the manifest_latest.zip
+    softlink.
+
+    :param manifest: Specify the manifest file path.
+    :param prefix_dir: The prefix directory to create the temp file.
+    """
+    if manifest:
```
Comment (elyezer) [inline automation_tools/__init__.py]: The following is redundant

```
if manifest:
    manifest = manifest
```

Just check for other cases.

File: automation_tools/__init__.py [hunk 2]
```diff
@@ -2532,3 +2538,20 @@ def download_manifest(url=None, session=None, distributor=None):
...
+        manifest = manifest
+    elif (prefix_dir is None and manifest is None):
+        manifest = download_manifest(prefix_dir='/opt/manifests/')
+    else:
+        manifest = download_manifest(prefix_dir=prefix_dir)
+    run('unlink /opt/manifests/manifest-latest.zip')
```
Comment (elyezer) [inline automation_tools/__init__.py]: If I specify a prefix, that means that it should be used here too, isn't it?

File: automation_tools/__init__.py [hunk 3]
```diff
@@ -2532,3 +2538,20 @@ def download_manifest(url=None, session=None, distributor=None):
...
+    run('ln -s {0} /opt/manifests/manifest-latest.zip'.format(manifest))
```
Comment (elyezer) [inline automation_tools/__init__.py]: Should be used the prefix?

**Comment by kbidarkar** [comment]
I at times re-factor the code at last moment and it becomes funny looking code. :)

**Comment by rplevka** [comment]
@kbidarkar thanks for addressing our comments,
ACK

**Comment by kbidarkar** [comment]
So here is the tested code and it was able to create the manifest on the desired server.
```
[xyz@abc.redhat.com] run: mv /tmp/tmp.bEPLLWgp5m.zip /opt/manifests/
[xyz@abc.redhat.com] run: chmod 644 /opt/manifests/tmp.bEPLLWgp5m.zip
[xyz@abc.redhat.com] run: unlink /opt/manifests/manifest-latest.zip
[xyz@abc.redhat.com] run: ln -s /opt/manifests/tmp.bEPLLWgp5m.zip /opt/manifests/manifest-latest.zip

Done.
Disconnecting from xyz@abc.redhat.com... done.
```

**Comment by sthirugn** [comment]
ACK

**Comment by elyezer** [comment]
ACK

**Comment by omaciel** [comment]
Just wondering if we should perform some type of assertion for `manifest_file = download_manifest()` to make sure that something valid was returned before moving and unlinking files around?

**Comment by rplevka** [comment]
@omaciel 
`download_manifest()` returns the attachment (in case the return header contains that):
https://github.com/SatelliteQE/automation-tools/blob/master/automation_tools/__init__.py#L2553
Otherwise it raises a `ValueError`.

**Comment by omaciel** [comment]
Thanks for the update @rplevka

**Pull request merged**